apps/admin-x-activitypub/src/components/search/ProfileSearchResultModal.tsx

```diff
@@ -31,6 +31,21 @@ const ProfileSearchResultModal: React.FC<ProfileSearchResultModalProps> = ({
         .filter(attachment => attachment.type === 'PropertyValue');
     const posts = profile.posts; // @TODO: Do any filtering / manipulation here
 
+    const [expanded, setExpanded] = useState(false);
```

Comment:
I think we would be better calling this `isExpanded` / `setIsExpanded` for consistency